### PR TITLE
Recompile regexes before comparing

### PR DESCRIPTION
**Pull request by fidr**

### CODE REVIEW

File: lib/walkman_server.ex [hunk 1]
```diff
@@ -94,4 +94,22 @@ defmodule WalkmanServer do
     Path.relative_to("test/fixtures/walkman", File.cwd!()) |> File.mkdir_p()
     filename(test_id) |> File.write!(:erlang.term_to_binary(Enum.reverse(tests)), [:write])
   end
+
+  defp args_match?({m, f, a}, {m2, f2, a2}) do
+    {m, f, normalize_args(a)} == {m2, f2, normalize_args(a2)}
+  end
+
+  defp normalize_args(args), do: Enum.map(args, fn arg -> normalize_arg(arg) end)
```
Comment (derekkraan) [inline lib/walkman_server.ex]: this is covered by the function head at line 108.

File: lib/walkman_server.ex [hunk 2]
```diff
@@ -94,4 +94,22 @@ defmodule WalkmanServer do
     Path.relative_to("test/fixtures/walkman", File.cwd!()) |> File.mkdir_p()
     filename(test_id) |> File.write!(:erlang.term_to_binary(Enum.reverse(tests)), [:write])
   end
+
+  defp args_match?({m, f, a}, {m2, f2, a2}) do
```
Comment (derekkraan) [inline lib/walkman_server.ex]: Actually you can also get rid of this one haha

Comment (fidr) [reply lib/walkman_server.ex]: Like this? You would still need the compare, or we can move that up to the place of the `args_match?` usage. 
```
  defp args_match?(args, args2) do
    normalize_args(args) == normalize_args(args2)
  end
```

Comment (derekkraan) [reply lib/walkman_server.ex]: I didn't mean like this (forgot about the compare) but this is good.

**Pull request merged**